src/plugins/query_enhancements/public/query_assist/utils/errors.ts

```diff
@@ -3,6 +3,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+// eslint-disable-next-line max-classes-per-file
 import { ResponseError } from '@opensearch-project/opensearch/lib/errors';
```

Comment:
nit: should we use the core type? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/core/server/http/router/response.ts#L42

Reply:
i think it should be the one from js library which has the `body` field. the core type gives a lot of lint errors since there's no `body` field https://github.com/opensearch-project/OpenSearch-Dashboards/blob/84d992ab0be48ef037b2c69baa5cfc9e77d64dd1/src/core/server/http/router/response.ts#L42-L48

and core type is in server, i don't think we can import it here